swagger.yaml

```diff
@@ -197,21 +190,26 @@ definitions:
       self:
         type: string
```

Comment:
Can you also add `format: url` to this field?

Reply:
Done